### ISSUE TITLE
Windows could be done more blizzlike.

### ISSUE THREAD

**Issue by Akumies** [Hoizame/AtlasLootClassic#11]
Hello!

Great addon, only thing what bothers me that it could be more like "Dungeon Journal" in retail wow.

Think about it :)

Thanks for the great addon!

**Comment by Cleric670** [comment]
Agree, the interface clashes pretty badly with the game UI. Having an option to reskin it to look like it actually BELONGS would be nice.

**Comment by Hoizame** [comment]
Hi this is planed
